lib/error-format.ts

```diff
@@ -0,0 +1,33 @@
+export function formatCallGraphError(error: Error): string {
+  if (error.message === 'Could not find target folder') {
+    return 'Couldn\'t scan for reachable vulns, to do so your code should be compiled, please run `mvn compile`.';
+  }
+  if (error.message === 'No entrypoints found') {
+    return 'Couldn\'t scan for reachable vulns, didn\'t find any entry point to your application.';
```

Comment:
Failed to scan fro reachable vulns. Application's entry point has not been found.

Reply:
How about: "Failed to scan for reachable vulns. Couldn't find the application entry point"?